designs/tutorial.stanza

```diff
@@ -1,81 +1,90 @@
+  val inherent-variance = tol+%(refVoltage)
   if inherent-variance > tol+%(output-voltage):
-    fatal("Target regulator accuracy infeasible with LM317A %_ > %_" % [inherent-variance tol+%(output-voltage)])
+    fatal("Target design accuracy infeasible with LM317A %_ > %_" % [inherent-variance tol+%(output-voltage)])
 
   ; Budget variability for the voltage divider
   val target-variance = tol+%(output-voltage) - inherent-variance
   ; 5mA of voltage-divider current to compensate for regulator adj current
   val divider-current = 5.0e-3
+  val adj-current = property(lm317a.adj-current) ; retrieve the property from the component definition
   ; Estimate offset voltage caused by current sourced from adj pin
-  val est-v-adj-offset = (typ-value(output-voltage) - typ-value(property(lm317a.reference-voltage))) / divider-current * property(lm317a.adj-current)
+  val est-v-adj-offset = (typ-value(output-voltage) - typ-value(refVoltage)) / divider-current * adj-current
 
   inst output-voltage-divider : ocdb/modules/passive-circuits/voltage-divider(high-voltage, adj-voltage, current) where :
     val high-voltage = tol%(center-value(output-voltage) - est-v-adj-offset, 0.0)
-    val adj-voltage = tol%(typ-value(property(lm317a.reference-voltage)),target-variance)
+    val adj-voltage = tol%(typ-value(refVoltage),target-variance)
     val current = divider-current                                                
 
-  net (output-voltage-divider.in lm317a.output)
-  net (output-voltage-divider.out lm317a.adj adj)
-  net (output-voltage-divider.lo gnd)
+  net (output-voltage-divider.in, lm317a.output)
+  net (output-voltage-divider.out, lm317a.adj adj)
+  net (output-voltage-divider.lo, gnd)
+  ; inside pcb-module lm317a-regulator 
   schematic-group(self) = lm317a
 
   ; Calculate the actual offset now that we know the optimized value of the resistors
-  val true-adj-offset = property(lm317a.adj-current) * (property(output-voltage-divider.r-lo.resistance) as Double)
+  val true-adj-offset = adj-current * (property(output-voltage-divider.r-lo.resistance) as Double)
   val vref = property(output-voltage-divider.output-voltage)
 
   ; Calculate the possible min and max values for the output voltage
-  val out = min-typ-max(min-value(property(lm317a.reference-voltage)) * typ-value(output-voltage) / max-value(vref) + true-adj-offset,
-                        typ-value(property(lm317a.reference-voltage)) * typ-value(output-voltage) / typ-value(vref) + true-adj-offset,
-                        max-value(property(lm317a.reference-voltage)) * typ-value(output-voltage) / min-value(vref) + true-adj-offset)
+  val out = min-typ-max(min-value(refVoltage) * typ-value(output-voltage) / max-value(vref) + true-adj-offset,
+                        typ-value(refVoltage) * typ-value(output-voltage) / typ-value(vref) + true-adj-offset,
+                        max-value(refVoltage) * typ-value(output-voltage) / min-value(vref) + true-adj-offset)
```

Comment:
`ref-voltage` instead of `refVoltage` (stanza prefers `kebab-case` to `camelCase`)

Reply:
done